### PR TITLE
r_with_python

### PR DESCRIPTION
**Pull request by WayneFd**
submission of r_with_python file

### CODE REVIEW

File: r_with_python.Rmd [hunk 1]
```diff
@@ -0,0 +1,208 @@
+# Integrate R with Python
+
+Weiwei Jiang
+
+## 1.Introduction
```
Comment (alexnsaunders) [inline r_with_python.Rmd:5]: No numbers after hashtags please

File: r_with_python.Rmd [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+## 1.Introduction 
+
+**Python** and **R** are important tools in Data Science. Sometimes you may be more familiar with a problem solution in one tool rather than the other. Sometimes you may have already developed a well-defined pipeline model or data analytic procedure in one of *Python* or *R*. It's tedious and time-consuming to rewrite the code in another form. Thus, it will bring lots of benefits if we can combine *Python* and *R*. This documents will foucs on how to run python code in R environment. There are  packages enable you to run R scripts in Python environment. rpy2 is one of the packages that are still keeping updating and the official documents will be a good start tutorial, as I put the link in Reference.
+
+## 2. Runing Python code in R scripts
+
+If you are searching a way to call *Python* from *R*, there exists an convenient CRAN package called "reticulate". This package enables you to easily transform between *R* and *Python* data type,using library from *Python*, source *Python* scripts and switch between different versions of *Python*.
+
+### 2.1 Installation
```
Comment (alexnsaunders) [inline r_with_python.Rmd:13]: number after hashtag - many examples in your files..

**Review by alexnsaunders — CHANGES_REQUESTED** [review]

**Comment by alexnsaunders** [comment]
thanks 🎉

**Comment by jtr13** [comment]
The build fails for this chapter since we don't have Python set up to run. I set eval=FALSE to get around this issue. We may be able to fix it, will discuss with collaborators.

**Comment by alexnsaunders** [comment]
https://jtr13.github.io/cc21fall2/integrate-r-with-python.html Here is your chapter congrats!

**Pull request merged**